src/cpp/rstudio-tests.in

```diff
@@ -33,6 +33,12 @@ findLibSegFault()
 
 runWatchdogProcess()
 {
+   if [[ "$OSTYPE" == "darwin"* ]]; then
+      # requires brew install coreutils
+      TIMEOUT=gtimeout
+   else
+      TIMEOUT=timeout
```

Comment:
I know these variables aren't case sensitive but it still feels weird to have one named `TIMEOUT` and one named `timeout`!

Reply:
Tweaked to be TIMEOUT_CMD instead.